### ISSUE TITLE
Can't run `pandablocks --help` without `softioc`

### ISSUE THREAD

**Issue by beenje** [PandABlocks/PandABlocks-client#36]
Hello,

Trying to update pandablocks to 0.3.0, I get:

```
pandablocks --help
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/pandablocks_1666966725075/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/bin/pandablocks", line 8, in <module>
    from pandablocks.cli import pandablocks
  File "/home/conda/feedstock_root/build_artifacts/pandablocks_1666966725075/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.11/site-packages/pandablocks/cli.py", line 13, in <module>
    from pandablocks.ioc import create_softioc
  File "/home/conda/feedstock_root/build_artifacts/pandablocks_1666966725075/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.11/site-packages/pandablocks/ioc/__init__.py", line 1, in <module>
    from .ioc import create_softioc
  File "/home/conda/feedstock_root/build_artifacts/pandablocks_1666966725075/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.11/site-packages/pandablocks/ioc/ioc.py", line 11, in <module>
    from softioc import alarm, asyncio_dispatcher, builder, fields, softioc
ModuleNotFoundError: No module named 'softioc'
```

`softioc` is supposed to be an extra requirement. Only the `softioc` command should require that package to be installed.
It should be possible to use the other commands without this requirement.

**Comment by AlexanderWells-diamond** [comment]
Thank you for pointing this out. I've released version `0.3.1` to fix this.

**Comment by beenje** [comment]
Thanks!
